tests/item_pocket_test.cpp

```diff
@@ -27,7 +27,6 @@
 
 static const ammotype ammo_test_9mm( "test_9mm" );
 static const item_pocket::pocket_type pocket_container = item_pocket::pocket_type::CONTAINER;
```

Comment:
You actually need to move this line below the itype_id block. The error message for that isn't very good.

Reply:
I'm not too sure which line you're referring to, `ammotype` or `item_pocket::pocket_type`?
Doesn't help that clang-tidy locally gives me way different errors.

Reply:
You picked the correct one.

Things like `ammotype` and `itype_id` are `string_id`s and they're required to be defined at the top in alphabetical order (first types alphabetical, then within a type the name alphabetical). The `pocket_type` is just an enum, so it's considered a disturbance there. The error messages really don't help in cases like this if you're not familiar with what's expected.

Dunno about local clang-tidy. Someone on discord might be able to really explain what you can or have to do about that. You can also ask about getting the tests approved there. Since this is your first PR, stuff can't start automatically.